rust/src/utils.rs

```diff
@@ -64,6 +64,86 @@ macro_rules! to_from_bytes {
     }
 }
 
+#[wasm_bindgen]
+#[derive(Clone, Debug)]
+pub struct TransactionUnspentOutput {
+    input: TransactionInput,
+    value: Value,
```

Comment:
Is there a reason to specify `value` and `address` instead of just using `output: TransactionOutput`?

Reply:
I oversaw that with **`TransactionOutput`**, your solution makes more sense right.

Reply:
I updated the CIP with this change using the cddl
```
transaction_unspent_output = [
        input: transaction_input,
        output: transaction_output,
]
```